### PR TITLE
First Module Capstone Project Submission

### PR DESCRIPTION
**Pull request by aeh1707**
This is a responsive website for a neurosience conference. Images and icons are added to suit the subject theme. The keynote speakers section of the home page is generated dynamically in javascript and the mobile menu feature is also implemented.

This is a demo video that talks shows the website and and its features: Check out a demo explanation about this project here [Live Video Demo Explainer](https://www.loom.com/share/b6004e57094a4b9d8d0d3fedc7748026)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+  list-style-type: none;
+}
+
+.fb {
+  width: 16px;
+}
+
+.tw {
+  width: 25px;
+}
```
Comment (zieeco) [inline styles.css]: - Please, kindly reduce the width to match the design below:

**your design**

weight



**original design**

theweight

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+    display: flex;
+    width: 300px;
+    gap: 15px;
+  }
+
+  .navbar-2 {
+    display: flex;
+    justify-content: space-around;
+    padding: 10px 0;
+  }
```
Comment (zieeco) [inline styles.css:545]: - [x]  Please, kindly reduce the spaces to match the design as much as possible below:
- [x] Also, add the remaining nav items to match the design. Remember you're making a clone of an existing website so you should try as much as possible to add what you see except the content that is meant to be personalized.
- [x] Kindly add some more padding to your register button to match the design's button below:
- [x] KIndly increase the spaces between each nav item to match the design as much as you can below:
- [x] Kindly add the active color to the `About` when a user clicks on it to indicate that the user is on the about page to improve the user experience.

**original design**

padding and missing

**your design**
misign and padding

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,785 @@
...
+h1 + p,
+hr + p.about {
+  padding: 10px;
+}
+
+h1 + p.home {
+  background-color: #d3d3d3;
+  border: 2px solid white;
+}
+
```
Comment (zieeco) [inline styles.css]: - Please, kindly use the correct background color as in the design which is #f7f7f7 to match the design below: 


**your background color**
cc3

**original background color**
ccolor

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,785 @@
...
+    font-size: 4rem;
+  }
+
+  h1 + p {
+    max-width: 600px;
+  }
+
+  p.time {
+    font-size: 2rem;
+  }
```
Comment (zieeco) [inline styles.css:596]: - Please let's use the correct color which is #515151: to match the design as much as possible below:

**your color**
dark

**design color**
dark2

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,785 @@
...
+
+  .speaker p {
+    margin: 10px 0;
+    width: 200px;
+  }
+
+  p.time + p {
+    font-size: 1.5rem;
+  }
```
Comment (zieeco) [inline styles.css]: - Please, let's use the correct font size on the second content and space between the two contents to match the design as much as possible below:


**your font size on second content**
f-s

**foriginal font size  and space on second content**
f-se

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,785 @@
...
+
+  .intro {
+    background-image: url('graphics/bg-desktop.png');
+    min-height: 60vw;
+  }
```
Comment (zieeco) [inline styles.css]: - Please, try to reduce the min-width to match the design as now the space before the `our goal` Is too much and doesn't match the design yet below:


**design space**
spax

**your space**
space

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,785 @@
...
+.intro p {
+  margin: 0.5rem;
+}
+
+.intro {
+  background-image: url('graphics/bg.png');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  min-height: 120vw;
+}
```
Comment (zieeco) [inline styles.css:223]: - Please, try to reduce the min-width to match the design as now the space before the `our goal` Is too much and doesn't match the design yet below:


**design space**
spax

**your space**
space

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,785 @@
...
+  .goals img {
+    width: 5rem;
+  }
+
+  .goals-container {
+    display: flex;
+    justify-content: center;
+    margin: 2rem 0;
+  }
+
```
Comment (zieeco) [inline styles.css:660]: - Please, kindly add the remaining card and reduce the spaces to match the design as yours is not complete yet below:
- Please, use the same background color for each card according to the design.
- Reduce the height of each cards according to the design.
- Please, ensure the content on the cards is clear enough to improve the user experience.
- Please, kindly use the same background color of the parent container to match the design as much as you can

**your card and space**
complete

**orighnal card and space**
completee

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,785 @@
...
+.agenda a {
+  display: inline-block;
+  color: #f2f1f1;
+  text-decoration: none;
+  font-size: 4.5vw;
+  text-align: center;
+  background-color: #ec5242;
+  padding: 4vw;
+  width: 100%;
+}
```
Comment (zieeco) [inline styles.css]: - Hey, you're doing great but please bear in mind that you're making a clone of an existing website. we should try making a complete clone as much as possible except for the content we're allowed to be personalized. Please kindly follow the design pattern on this element according to the design below:

**your implementation**
replace

**design inplementation**
rep

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,785 @@
...
+  }
+
+  .speaker-content {
+    width: 200px;
+  }
+
+  .speaker-img {
+    width: 100px;
+  }
+
```
Comment (zieeco) [inline styles.css]: - Please, kindly add another speaker to match the design as yours is 5 remain one to complete 6
- Please, kindly reduce the margin-left and margin-right as your's is in the center but on the design, it's not in the center
- Please, kindly use the same color according to the design as much as you can

**6**
kfea

**5**
keynote

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,785 @@
...
+.organizers {
+  background-color: #272a31;
+  padding: 30px 30px 50px;
+}
+
+.organizers img {
+  width: 150px;
+}
+
```
Comment (zieeco) [inline styles.css]: - [x] Please let's stick to the background color according to the design as much as possible.
- [x] Please let's stick to the color of each logo by adding opacity according to the design as much as possible.
- [x] Please let's stick to the width of each logo size according to the design as much as possible.
- [x] Please kindly reduce the padding of the partner's container to match the design as much as we can

**your design**
widthhh

**original design**
widthh

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,785 @@
...
+  ************************** */
+
+  .footer img {
+    width: 200px;
+  }
+
+  .home-footer-container {
+    justify-content: center;
+    align-items: center;
+  }
```
Comment (zieeco) [inline styles.css:577]: - Please let's use the right color of the footer element as much as we can to match the design below:


**original design**
collll

**your design**
colll

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,785 @@
...
+
+  /* Intro Section Styles
+  ************************** */
+
+  .intro-subheading {
+    font-size: 3rem;
+  }
```
Comment (zieeco) [inline styles.css:584]: - [ ] Please, kindly remove this text content as it's not part of the about page in the original design below:

**original about page**
here

**you about page**
r

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,785 @@
...
+  /* Intro About Section Styles
+  ************************** */
+
+  .intro.about .intro-container {
+    align-items: center;
+  }
+
+  hr + p.about {
+    width: 60vw;
+  }
```
Comment (zieeco) [inline styles.css:638]: - Please, let's use the correct background color as used in the design as much as we can below: 

**your background color**
bk


**descign background color**
bk2

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,785 @@
...
+  gap: 4vw;
+}
+
+/* Our Team Section Styles
+************************** */
+
+.team {
+  padding: 30px 30px 50px;
+}
+
```
Comment (zieeco) [inline styles.css:389]: - [ ] Please, let's update the team section by adding more content following the design pattern according to the design as much as we can below:

**original section implementation pattern**
hett


**your design implementation pattern**
designn

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,785 @@
...
+  color: #272a31;
+}
+
+/* About Footer Section Styles
+*********************************/
+
+.footer.about {
+  background: #272a31;
+  color: #f2f1f1;
+}
```
Comment (zieeco) [inline styles.css]: - Please let's use the original background color according to the design which is #2E2E2E

File: index.html [hunk 17]
```diff
@@ -0,0 +1,168 @@
...
+                        <img class="speaker-img" src="src" alt="">
+                    </div>
+                    <div class="speaker-content">
+                        <h3></h3>
+                        <div class="speaker-subheading"></div>
+                        <hr>
+                        <p></p>
+                    </div>
+                </article>
+            </div>
```
Comment (zieeco) [inline index.html]: - [ ] Hey, the speaker section should be added dynamically using javascript according to the project requirement. Please, kindly update these changes to meet the project requirement :pray:

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @aeh1707 


Great work on the capstone project 👏 
You've done well implementing the project, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — APPROVED** [review]
Hi @aeh1707, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**